modules/schematics/src/entity/files/__name@dasherize@if-flat__/__name@dasherize@group-actions__.actions.ts.template

```diff
@@ -1,53 +1,20 @@
-import { createAction, props } from '@ngrx/store';
+import { createActionGroup, emptyProps, props } from '@ngrx/store';
 import { Update } from '@ngrx/entity';
 
 import { <%= classify(name) %> } from '<%= featurePath(group, flat, "models", dasherize(name)) %><%= dasherize(name) %>.model';
 
-export const load<%= classify(name) %>s = createAction(
-  '[<%= classify(name) %>/API] Load <%= classify(name) %>s', 
-  props<{ <%= camelize(name) %>s: <%= classify(name) %>[] }>()
-);
-
-export const add<%= classify(name) %> = createAction(
-  '[<%= classify(name) %>/API] Add <%= classify(name) %>',
-  props<{ <%= camelize(name) %>: <%= classify(name) %> }>()
-);
-
-export const upsert<%= classify(name) %> = createAction(
-  '[<%= classify(name) %>/API] Upsert <%= classify(name) %>',
-  props<{ <%= camelize(name) %>: <%= classify(name) %> }>()
-);
-
-export const add<%= classify(name) %>s = createAction(
-  '[<%= classify(name) %>/API] Add <%= classify(name) %>s',
-  props<{ <%= camelize(name) %>s: <%= classify(name) %>[] }>()
-);
-
-export const upsert<%= classify(name) %>s = createAction(
-  '[<%= classify(name) %>/API] Upsert <%= classify(name) %>s',
-  props<{ <%= camelize(name) %>s: <%= classify(name) %>[] }>()
-);
-
-export const update<%= classify(name) %> = createAction(
-  '[<%= classify(name) %>/API] Update <%= classify(name) %>',
-  props<{ <%= camelize(name) %>: Update<<%= classify(name) %>> }>()
-);
-
-export const update<%= classify(name) %>s = createAction(
-  '[<%= classify(name) %>/API] Update <%= classify(name) %>s',
-  props<{ <%= camelize(name) %>s: Update<<%= classify(name) %>>[] }>()
-);
-
-export const delete<%= classify(name) %> = createAction(
-  '[<%= classify(name) %>/API] Delete <%= classify(name) %>',
-  props<{ id: string }>()
-);
-
-export const delete<%= classify(name) %>s = createAction(
-  '[<%= classify(name) %>/API] Delete <%= classify(name) %>s',
-  props<{ ids: string[] }>()
-);
-
-export const clear<%= classify(name) %>s = createAction(
-  '[<%= classify(name) %>/API] Clear <%= classify(name) %>s'
-);
+export const <%= classify(name) %>Actions = createActionGroup({
```

Comment:
Should action groups be an option to the schematic?
I do think some teams prefer single actions because you can use some refactoring tools with them, which you can't with groups.

Reply:
I see your point. But in my opinion it reducers duplication and implements a namespace for the action. This outweights the keeping of the status-quo. 
I was also pretty surprised that something like `createActionGroup` exist and it isn't the default. Because a lot of people don't know about this feature and the main critisism against ngrx is the boilerplate which this feature reduces.
We could either do a `--legacy`- or a `--verbose`-flag or create a new schematics, but maintaining 2 variants is time-consuming. Maybve you have a way to ask different usergroups? For me so far everybody I showed this liked the new method more than the old one.

Reply:
Sorry for my delayed answer.

AFAIK we don't have a way to ask different usergroups.

I do think that having this as a default is good, the only question that remains open is if we should provide a flag which keeps the current implementation.
After giving this some more thought I think we can keep this PR as is, we could revisit this in the feature if it's really needed.

Is this ok for you @brandonroberts and @markostanimirovic ?

Reply:
I think it's reasonable to move forward without it being behind a flag. We can mark it as breaking for v16 so people are aware of it

Reply:
Sounds good to me 👍